tests/unit/test_server_mock_conductor.js

```diff
@@ -280,391 +285,391 @@ describe("Server with mock Conductor", () => {
     } finally {}
   });
 
-  it("should forward signal from conductor to client", async () => {
```

Comment:
why is this all commented out?

Reply:
I think it was commented out in `develop`. Just merged develop in and it is commented back in